### ISSUE TITLE
SOS：loss.backward()和self.optimizer.step()

### ISSUE THREAD

**Issue by Gshanyoufushuxianshen** [RUCAIBox/RecBole#2072]
背景：我是在复现一篇名为“AFDGCF”的推荐系统方面的论文，它使用了Recbole这一先进的框架进行编写代码。我现在的目的是想找到它的loss函数关于use embedding和item embedding 的计算偏导的代码式子，这样我才可以知道参数是如何进行更新的。并后续尝试替换为自己的式子。文章代码如下https://github.com/U-rara/AFDGCF.
问题描述：
在_train_epoch函数中，会调用loss.backward()和self.optimizer.step()，目的应该是计算损失函数的梯度并更新。可是当点击进去，我感到困惑，一方面在于它有太多的选项了。
我所知道的是它使用的是adam优化器，但是就在loss.backward（）我也无法在几十个中找到需要的选项。
![Snipaste_2024-08-10_16-15-56](https://github.com/user-attachments/assets/7bb40bb2-1bf1-4a46-b66e-9f84241dc405)
![Snipaste_2024-08-10_16-18-19](https://github.com/user-attachments/assets/facb836d-2fa1-45c1-9bd0-c55381841147)
最后：对于一个大二的科研小白，这真的是一个很难以解决的问题，寻找不到身边的人帮助。非常感谢你能看到这里，如果你能给我提一点建议或者帮助，我将感激不尽，就当是在帮助曾经的自己吧。如果你和我一样是小白，并且也在科研的道路上前行，我非常愿意我们可以交个朋友，共同学习。
